docs/UserGuide.md

```diff
@@ -97,11 +97,10 @@ confidence to make full use of TAHelper's features.
     - Mac OS:
       - Search for Terminal in "Utilities" under "Applications".
       - It should look something like this:
-      
 
 5. Launching TAHelper
     - Type `java -jar tahelper.jar` command and hit Enter to run TAHelper.<br>
-    - It should look something like this (in this case my jar file is in a folder called tahelper): 
+    - It should look something like this (in this case my jar file is in a folder called `tahelper`):
     - ![cmd](images/cmdwinguide2.png)
     - A GUI similar to the below should appear in a few seconds.<br>
     - ![Ui](images/Ui.png) (to update!!).
```

Comment:
I think we should remove the bullet points from the images since it was an issue brought up from PED

Reply:
Yep @mahadhir247 no worries on this. I will update together with the UI in a seperate PR